course/templatetags/course.py

```diff
@@ -81,6 +81,11 @@ def is_open(entry, now):
     return entry['opening_time'] <= now <= entry['closing_time']
 
 
+@register.filter
+def exercises_open(entry, now):
```

Comment:
In this file, function `exercises_open` looks identical to `is_open`. Are both needed?

Reply:
In the course module class, `is_open` uses the reading time, so now the `is_open` template tag is different when it uses the opening time. Inconsistency causes confusion.

Reply:
The method `is_open` has been removed.